pyproject.toml

```
@@ -89,7 +89,7 @@ boto3 = [ "boto3~=1.35" ]
 deluge = [ "deluge-client~=1.10" ]
 ftp = [ "ftputil~=5.1" ]
 matrix = [ "matrix-nio~=0.25.2" ]
-plexapi = [ "plexapi==4.17.0" ]
+plexapi = [ "plexapi==4.17.1" ]
```

```suggestion
plexapi = [ "plexapi~=4.17" ]
```
There’s just one small issue. FlexGet currently pins dependencies only to the major version rather than fully locking the version, in order to avoid Renovate creating too many PRs.

---

Good. That really belongs only in the .lock file IMHO. Happy to fix that.